src/main/java/com/gluonhq/richtext/RichTextAreaSkin.java

```diff
@@ -85,8 +85,8 @@
     private final Map<KeyCombination, ActionBuilder> INPUT_MAP = Map.ofEntries(
         entry( new KeyCodeCombination(RIGHT, SHIFT_ANY, ALT_ANY, CONTROL_ANY, SHORTCUT_ANY), e -> ACTION_CMD_FACTORY.caretMove(Direction.FORWARD, e)),
         entry( new KeyCodeCombination(LEFT,  SHIFT_ANY, ALT_ANY, CONTROL_ANY, SHORTCUT_ANY), e -> ACTION_CMD_FACTORY.caretMove(Direction.BACK, e)),
-        entry( new KeyCodeCombination(DOWN,  SHIFT_ANY),                                     e -> ACTION_CMD_FACTORY.caretMove(Direction.DOWN, e.isShiftDown(), false, false)),
-        entry( new KeyCodeCombination(UP,    SHIFT_ANY),                                     e -> ACTION_CMD_FACTORY.caretMove(Direction.UP, e.isShiftDown(), false, false)),
+        entry( new KeyCodeCombination(DOWN,  SHIFT_ANY, ALT_ANY, SHORTCUT_ANY),              e -> ACTION_CMD_FACTORY.caretMove(Direction.DOWN, e)),
+        entry( new KeyCodeCombination(UP,    SHIFT_ANY, ALT_ANY, SHORTCUT_ANY),              e -> ACTION_CMD_FACTORY.caretMove(Direction.UP, e)),
```

Comment:
On Windows Notepad, pressing `DOWN` and `UP` doesn't extend selection.

Reply:
I'm following TextArea settings (TextAreaBehavior for both Mac and Windows).

For a TextArea on Windows, Ctrl+ Up/Down goes to beginning/end of paragraph, Ctrl+Shift+ Up/Down, selects from current position to beginning/end of paragraph.